src/main/java/com/scalar/admin/Command.java

```diff
@@ -3,5 +3,6 @@
 public enum Command {
   PAUSE,
   UNPAUSE,
-  STATS
+  STATS,
+  CHECKPAUSED
```

Comment:
Sorry, it's a bit picky, but it should be `CHECK_PAUSED`.

Reply:
Sorry, my bad.
It's renamed in https://github.com/scalar-labs/scalar-admin/pull/10/commits/a557b584d7348230b228021f5b911d384b6e7570